pkg/cmd/pr/merge/merge.go

```diff
@@ -415,6 +438,12 @@ func (m *mergeContext) deleteRemoteBranch() error {
 	return m.infof("%s Deleted branch %s%s\n", m.cs.SuccessIconWithColor(m.cs.Red), m.cs.Cyan(m.pr.HeadRefName), branch)
 }
 
+// Add the Pull Request to a merge queue
+// Admins can bypass the queue and merge directly
+func (m *mergeContext) addToMergeQueue() bool {
```

Comment:
It doesn't seem like this method actually adds the PR to a merge queue, so I would suggest renaming it so that it's clear it's only a check and not any kind of action.
```suggestion
func (m *mergeContext) shouldAddToMergeQueue() bool {
```

Reply:
I wasn't sure about naming on this one, will update!